src/lib.rs

```diff
@@ -60,18 +60,23 @@ pub enum DisplayState {
     Editable,
 }
 
-/// Configure spacing and separators for the view
+///Controls the visual output of the `HexView` struct.
+///
+///There are various options which can be altered. For a detailed description of them, please see the fields below.
+///For the changes to apply, you need to use [`set_config`].
+///
+///[`set_config`]: struct.HexView.html#method.set_config
 #[derive(Debug, Clone, Copy)]
 pub struct HexViewConfig {
-    /// Number of bytes per line
+    /// Controls the number of bytes per line. It needs to be greater than 0 and equal or higher than the bytes_per group value. Default is 16
```

Comment:
Please write \`bytes_per_group\` so it will be linked automatically by rustdoc

Also a `.` is missing after 16